### PR TITLE
Task/readme

### PR DESCRIPTION
**Pull request by adael2**
#### What's this PR do?

Add Readme for the repository (For the documentation)

#### Where should the reviewer start?

Readme.md

#### How should this be manually tested?

Check the file Readme.md if it's ok.

#### Any background context you want to provide?

Check if there is more to add at this file.

This template was adapted ~retrived~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,91 @@
...
+## Test
+
+To test the application
+
+* In jp2_offline/config.js:
+
+```shell
+
+  DEBUG: false,
+  ENV: 'Testing'
```
Comment (Ral15) [inline README.md]: 'testing'**

File: README.md [hunk 2]
```diff
@@ -1 +1,91 @@
...
+  npm test test/your-test.js
+
+```
+
+* To create a test, in directory test there is a file called base-test.js You can implement your test with this little guide.
+
+* To use eslint:
+
+ ```shell
+  npm lint
```
Comment (Ral15) [inline README.md]: i believe it is npm run lint :)

File: README.md [hunk 3]
```diff
@@ -1 +1,91 @@
...
+
+```shell
+  npm build
+```
+
+This creates a executable for windows x64. To changes architecture and operating system you can run
+
+```shell
+  # To create the same executable
+  electron-packager . "San Juan Pablo II" --platform=win32 --arch=x64
```
Comment (Ral15) [inline README.md:50]: will this work on any OS? or just in Windows?

**Review by Ral15 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (adael2) [reply README.md:50]: Yes, but you need to install Wine. That's why I put the link to Electron packager for more information

**Review by Ral15 — APPROVED** [review]

**Review by erikiado — APPROVED** [review]

**Pull request merged**